extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -144,15 +144,15 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     // validateData, from 3p/3p/js, after moving it someplace common.
     const startTime = Date.now();
     const global = this.win;
-    let adClientId = this.element.getAttribute('data-ad-client');
+    let adClientId = this.getStringData('adClient');
```

Comment:
Can you create a linter rule for A4A code that ensures getAttribute is explicitly allowed to ensure you caught all of the cases and make sure we don't miss them going forward?